addons/website/static/tests/tours/link_tools.js

```diff
@@ -240,6 +266,7 @@ wTourUtils.registerWebsitePreviewTour('link_tools', {
     {
         content: "Change URL into an email address",
         trigger: "#o_link_dialog_url_input",
+        extra_trigger: "div#oe_snippets:not(div.o_we_ui_loading)",
```

Comment:
That seems most likely like a workaround to a possible race condition in our code @qsm-odoo , weird to have to add this everywhere in this tour.
From what I understood so far, the loading is about the autocomplete URL loading:
![image](https://github.com/odoo/odoo/assets/30048408/8a1b2436-59be-4b16-9a8a-d7e9f60d0368)

Seems very fragile to rely on that